source/scripts/index.js

```diff
@@ -0,0 +1,75 @@
+window.addEventListener('DOMContentLoaded', init);
+
+/**
+ * call all the functions after the DOM is loaded
+ */
+function init() {
+    //let nodes = getNotes();
+    let notes = getNotesFromStorage();
+    addNotesToDocument(notes);
+
+    initEventHandler();
+}
+
+/**
+ * For testing purpose, it will create a new note and save it to local storage
+ * Retrive all notes from local storage
+ * @returns either the notes json data or the empty list
+ */
+function getNotesFromStorage() {
+    const notes =  localStorage.getItem('notes');
+    return notes ? JSON.parse(notes) : [];
+}
+
+/**
+ * For testing purpose, it will save the notes to local storage
+ * Takes in an array of recipes, converts it to a string, and then
+ * saves that string to 'notes' in localStorage
+ * @param {Array<Object>} notes An array of notes
+ */
+function saveNotesToStorage(notes) {
+    localStorage.setItem('notes',JSON.stringify(notes));
+}
+
+/**
+ * append the list items to the list in the document
+ * @param {Object} notes containing all the notes in the local storage
+ */
+function addNotesToDocument(notes) {
+    let list = document.querySelector('.list');
+
+    notes.forEach(note => {
+        let listItem = document.createElement('list-item');
```

Comment:
Can we use a more descriptive name than 'list-item'? Maybe something like 'dashboard-row' might be a little more clear that this element is meant for the dashboard

Reply:
Should be good now